interview.md

```diff
@@ -11,41 +11,104 @@ _What need is met by your idea? Who are you helping? What is that person's
+
+At this moment, there are usually pre-existing file templates that people can copy and paste their ideas or apps that walk users through creating a document with some structure from scratch. However, neither of these solutions helps users sift through the information they already have - meaning they might potentially have forgotten or missed details that seem trivial but are actually important. 
+
+If I could help them, I want to give users a tool to do a coarse re-ordering of their documents so that those hidden connections may become more visible. I believe this tool should not be a substitute for those templates or apps mentioned earlier. Instead, users should be able to use this tool quickly so that the later parts of the planning process (whether for an itinerary, essay, or some other format) can be smoother.
+
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL for this scenario is appropriate because it can be an interface to quickly sift through the content without the fatigue from manual editing. A general API can also do the same, but since the aim is to automate a task in the writing process, a DSL can be more easily designed so that users do not have to context-switch to a “programming” style of thinking and helping people stay “in the zone” of writing.
+
+This need is domain-specific - it focuses on a subset of word documents that have an implicit or designed order and modules that can be identified by keywords. The limited expressiveness is also guaranteed because this task is focused on reordering existing data, rather than editing the data itself. 
+
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+I interviewed my high school friend (who works in sales/marketing) about her experience with computers. She recently booked a flight for a trip and ran into a lot of issues with booking flights. Our conversation then verged into planning for travel, which is how this idea came about.
+
+This idea is exciting because it involves balancing between “programming” for users that are likely not programmers with enough notation to prevent confusion (like the Perl vs AppleScript example from the last homework). It might also get my friend to be less fearful of breaking computer-related things and try writing scripts to help her tasks without feeling like she is “programming”.
+
```

Comment:
I definitely know people who feel the same way towards computer-related task, constantly in fear of breaking things. Out of all three of your topics, this one seems like it will reach the greatest audience. Its a good idea that fills a large niche, it just hasn't been done well yet